apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx

```diff
@@ -297,21 +304,22 @@ export const ModerationNotification = (props) => {
             </div>
           </div>
         </li>
+        {/* FIXME once userReport added to serializer */}
+        {numReports > 0 && !aiReport &&
+          <span className="badge a4-comments__badge a4-comments__badge--sug">
+            {translated.reportUser}
+          </span>}
 
-        {numReports > 0 &&
-          <li>
-            <p>
-              <i className="fas fa-exclamation-circle me-1" aria-hidden="true" />
-              {getLink(translatedReportText(numReports), commentUrl)}
-            </p>
-          </li>}
-        <li>
-          <p>{commentText}</p>
-        </li>
+        {aiReport &&
+          <span className="badge a4-comments__badge a4-comments__badge--not">
+            {translated.reportAi}
+          </span>}
+
+        <p>{commentText}</p>
 
         {aiReport &&
           <AiReportExplanation
-            AiReport={translated.dummyText}
+            AiReport={aiReport.explanation}
```

Comment:
Can we have this explanation in a separate div or p element from the introductory text, design has another paragraph for the actual AI text.
![image](https://github.com/liqd/a4-defakts/assets/5695572/e433c33f-b9e8-48de-b685-9230357e4328)
Now all blends
![image](https://github.com/liqd/a4-defakts/assets/5695572/8f62ac01-f1af-4905-98cd-24149a42f9b2)
and I see the story asks for Read more/ Show less instead of Read less

Reply:
should be fixed! :)